src/bootstrap/src/utils/tests/git.rs

```diff
@@ -129,6 +129,9 @@ impl GitCtx {
 
     fn git_cmd(&self) -> Command {
         let mut cmd = Command::new("git");
+        cmd.env("GIT_CONFIG_NOSYSTEM", "1");
+        cmd.env("GIT_CONFIG_SYSTEM", "/tmp/nonexistent");
+        cmd.env("GIT_CONFIG_GLOBAL", "/tmp/nonexistent");
```

Comment:
I hope we don't also have to override `$HOME` or `$XDG_USER_HOME` somehow (cf. https://git-scm.com/docs/git#Documentation/git.txt-codeGITCONFIGNOSYSTEMcode)